sql/share/errmsg-utf8.txt

```diff
@@ -6511,7 +6511,7 @@ ER_KEY_BASED_ON_GENERATED_VIRTUAL_COLUMN
 ER_WRONG_FK_OPTION_FOR_VIRTUAL_COLUMN
   eng "Cannot define foreign key with %s clause on a computed column"
 ER_WARNING_NON_DEFAULT_VALUE_FOR_VIRTUAL_COLUMN
-  eng "The value specified for computed column '%s' in table '%s' ignored"
+  eng "The value specified for computed column '%s' in table '%s' is ignored"
```

Comment:
I actually agree with your suggestion.  I simply used "is" because that is what the bug report said.  However, I agree that "has been ignored" is preferable to "is ignored".  I'm tempted to re-commit and re-push this, but am slightly afraid since this update already included some other commit (from Igor) that I don't know how it tangled up in my original commit.  Let me know if you think I should do this.

I had wiped that branch on my server, so I would have to start from scratch.  I'm envisioning:

git clone --depth=50 --branch=5.5 https://github.com/chriscalender/server
cd server
git checkout -b MDEV-17449
git rebase 5.5
<edit file>
git commit -am "Fix for MDEV-17449, typo in error message"
git push --set-upstream https://github.com/chriscalender/server/ MDEV-17449
git rebase -i HEAD~2

Reply:
You already have "broken" 5.5 in your repo. Try `git reset --hard HEAD^^` then git pull.